### PR TITLE
[v2.9] Move extensions into new actions package 

### PR DESCRIPTION
**Pull request by igomez06**
## Problem
 There were a lot extensions that were being maintained in a shepherd when it made more sense to maintain them directly in rancher/rancher
 
## Solution
 Move identified actions into the new package. 
  
### Automated Testing

* Test types added/modified:
    * Unit
    * Integration (Go Framework)
    * Integration (v2prov Framework)
    * Validation (Go Framework)
    * Other - Explain: _EXPLAIN_
    * None
    * _REMOVE NOT APPLICABLE BULLET POINTS ABOVE_
* If "None" - Reason: _EXPLAIN THE REASON_
  
* If "None" - GH Issue/PR: _LINK TO GH ISSUE/PR TO ADD TESTS_

Summary: _TODO_

### Regressions Considerations

_TODO_

Existing / newly added automated tests that provide evidence there are no regressions:
* _TODO_

### CODE REVIEW

**Review by caliskanugur — APPROVED** [review]
Looks good to me

**Review by markusewalker — APPROVED** [review]

File: go.mod
Comment (slickwarren) [inline go.mod:17]: this still needs to be updated now that shepherd has merged, waiting on this for approval.

**Review by caliskanugur — APPROVED** [review]

File: tests/v2/actions/rbac/rbac.go
Comment (slickwarren) [inline tests/v2/actions/rbac/rbac.go:10]: for those curious, this is one file that had a line count difference compared to main. looks fine but not sure why the files are so different overall

**Review by slickwarren — DISMISSED** [review]

**Review by anupama2501 — APPROVED** [review]

**Review by markusewalker — DISMISSED** [review]

**Review by slickwarren — APPROVED** [review]

**Review by markusewalker — APPROVED** [review]

**Pull request merged**
